ozaria/site/components/interactive/PageInteractive/BaseDraggableSlot.vue

```diff
@@ -0,0 +1,135 @@
+<script>
```

Comment:
I was writing some comments on the props of this file, and then realized that it all made lots of sense.
This file looks good to me.
(Leaving a note as it's a long pr)